### PR TITLE
45 ci inventory validation

### LINKED ISSUE #45: CI with inventory-example validation
We should create an action that runs the `validate` target on the inventory-example file (and maybe all the other examples?)

This should ensure that all our example and yaml documentations is aligned with any last changes in the code and assertion

### PR DESCRIPTION
**Pull request by 89luca89**
Fix #45

### CODE REVIEW

File: roles/terrible/.travis.yml [hunk 1]
```diff
@@ -26,4 +26,5 @@ script:
   - ansible-playbook tests/test.yml -i tests/inventory --syntax-check
```
Comment (alegrey91) [inline roles/terrible/.travis.yml:26]: Why did you add a .travis.yml file?
With the next issue of mine, this is going to be overridden.
Is this job also redundant?

Comment (89luca89) [reply roles/terrible/.travis.yml:26]: This is automatically generated from ansible-galaxy when we generated the basic role
